src/containers/App/App.tsx

```diff
@@ -13,13 +13,13 @@ export default class App extends React.PureComponent<
     const { hero, isHomePage, children, ...rest } = this.props
     const hasHero = hero != null && isHomePage
     return (
-      <>
+      <React.Fragment>
```

Comment:
Why this change?

Reply:
The only other piece of code I found using a Fragment used `React.Fragment` (TranslationProvider.tsx) so I went with that on ModalProvider and wanted to use the same thing everywhere

BUT

looking closer the README uses `<>` 😅 soo...which one do you like best? I'll update the PR only using one of the two. I have no preference

Reply:
I would rather use `<>`, i think the marketplace is the only one that still uses `React.Fragment` but everywhere else i think we are using the short version (i like it better 😇 ).

Reply:
<> it is 🚂 !